filament/src/VertexBuffer.cpp

```diff
@@ -256,14 +256,16 @@ void FVertexBuffer::setBufferObjectAt(FEngine& engine, uint8_t bufferIndex,
 }
 
 void FVertexBuffer::setExternalBufferAt(FEngine& engine, uint8_t bufferIndex,
-         void* externalBuffer) {
+         intptr_t externalBuffer) {
     ASSERT_PRECONDITION(!mBufferObjectsEnabled, "Please use setBufferObjectAt()");
     ASSERT_PRECONDITION(mExternalBuffersEnabled, "Please use setBufferAt()");
     if (bufferIndex < mBufferCount) {
         assert_invariant(mBufferObjects[bufferIndex]);
+        engine.getDriverApi().setupExternalResource(externalBuffer);
```

Comment:
Same here, the docs of `filament::VertexBuffer::setExternalBufferAt()` needs an update

Reply:
Docs are updated in this commit: 717d16a